content/general/events.textile

```diff
@@ -0,0 +1,390 @@
+---
+title: Reactor Events
+section: general
+index: 20
+languages:
+  - none
+jump_to:
+  Help with:
+    - Available integrations#integrations
+    - Configuring a Webhook#configure
+    - Sources#sources
+    - Single vs Batched requests#batching
+    - Envelopes#envelope
+    - Payload encoding#encoding
+    - Webhook Security#security
+    - Examples#examples
+---
+
+Reactor Events let you easily allow endpoints and services to be notified via an HTTP request following "messages being published":/realtime/messages or "presence events":/realtime/presence (such as members entering or leaving) on "channels":/realtime/channels.
+
+p(tip). Reactor Events are "rate limited":#transport and are suitable for low to medium volumes of updates. If you expect a high volume of events and messages (averaging more than 25 per second), then you should consider using our "message queues":/general/queues or "firehose":/general/firehose as they are more suitable for higher volumes.
+
+Generally, realtime interactions such as subscribing to events and messages are best done using our "realtime client libraries":/realtime, or our "message queues":/general/queues or "firehose":/general/firehose. However, it can sometimes be preferable to be notified over HTTP when events occur such as a user publishing a message or members entering the presence set on a channel. For example, you may want to start pushing content to a channel only when the channel becomes active and there is someone there to receive the data, or they may want to send a welcome message to someone when they first enter a chat channel. Reactor Events are useful for achieving this in both these scenarios.
```

Comment:
"Generally, realtime interactions such as subscribing to events and messages are best done using our" is slightly misleading.

"For clients subscribing to events and messages on-demand, this is often best done using our "realtime client libraries":/realtime or subscribing to Ably using any of the "realtime protocols we support":https://www.ably.io/adapters.  However, when a persistent subscription is required to push data into third party systems, the Reactor is designed for this use case and is available as Reactor Events (for HTTP requests, serverless functions, etc), Reactor Queues (data is pushed into our own hosted message queues that you can subscribe to), or Reactor Firehose (stream events into third party systems like Kafka and AWS Kinesis).  

If you want to be notified as events arise, and would like to trigger serverless functions, or invoke an HTTP request to an endpoint, then Reactor Events is the right choice.  For example, if you want to send a welcome message to someone when they become present on a chat channel, you can use Reactor Events to trigger a serverless function immediately after they enter, which in turn can publish a welcome message back to that user on the chat channel.

---

WDYT?  

You also mention things like "you may want to start pushing content to a channel only when the channel becomes active and there is someone there to receive the data", yet we don't mention channel lifecycle events  and we only support in batched mode.  Happy to include, but previously you only mentioned messages and presence as the source of events.